agent/consul/acl.go

```diff
@@ -237,7 +272,8 @@ type ACLResolverConfig struct {
 //   upon.
 //
 type ACLResolver struct {
-	config *Config
+	// TODO: store the ACLResolverConfig as a field instead of copying all the fields onto ACLResolver.
```

Comment:
```suggestion
	// TODO: store the ACLResolverSettings as a field instead of copying all the fields onto ACLResolver.
```

Reply:
Aren't you already doing what the comment says to do?

Reply:
oops, yes this is done. The comment was from an earlier revision and I forgot to remove it. Will remove the comment now.